eduVPN/ConnectionService.swift

```diff
@@ -261,40 +233,61 @@ class ConnectionService: NSObject {
             handler(.failure(Error.noHelperConnection))
             return
         }
-
+        
         let bundle = Bundle.init(for: ConnectionService.self)
         let openvpnURL = bundle.url(forResource: "openvpn", withExtension: nil, subdirectory: ConnectionService.openVPNSubdirectory)!
         let upScript = bundle.url(forResource: "client.up.eduvpn", withExtension: "sh", subdirectory: ConnectionService.openVPNSubdirectory)!
         let downScript = bundle.url(forResource: "client.down.eduvpn", withExtension: "sh", subdirectory: ConnectionService.openVPNSubdirectory)!
-        let leasewatchPlist = URL(fileURLWithPath: "/Library/Application Support/eduVPN/LeaseWatch.plist")
-        let leasewatchScript = bundle.url(forResource: "leasewatch", withExtension: "sh", subdirectory: ConnectionService.openVPNSubdirectory)!
         var scriptOptions = [
             "-6" /* ARG_ENABLE_IPV6_ON_TAP */,
             "-f" /* ARG_FLUSH_DNS_CACHE */,
             "-o" /* ARG_OVERRIDE_MANUAL_NETWORK_SETTINGS */,
             "-r" /* ARG_RESET_PRIMARY_INTERFACE_ON_DISCONNECT */,
-            "-w" /* ARG_RESTORE_ON_WINS_RESET */,
-            "-m" /* ARG_MONITOR_NETWORK_CONFIGURATION */,
-            "-t" /* ARG_TB_PATH */,
-            bundle.bundlePath
+            "-w" /* ARG_RESTORE_ON_WINS_RESET */
         ]
         let developerMode = preferencesService.developerMode
         if developerMode {
             scriptOptions.append("-l" /* ARG_EXTRA_LOGGING */)
         }
-       
+        
         self.configURL = configURL
-        helper.startOpenVPN(at: openvpnURL, withConfig: configURL, upScript: upScript, downScript: downScript, leasewatchPlist: leasewatchPlist, leasewatchScript: leasewatchScript, scriptOptions: scriptOptions) { (success) in
-            if success {
+        
+        //        out=0 :Failed Successfull
+        //        out=1: Successful
+        //        out=2; Failed
+        
+        helper.startOpenVPN(at: openvpnURL, withConfig: configURL, upScript: upScript, downScript: downScript, scriptOptions: scriptOptions) { (status) in
+            
+            
+            
+            var data:Array? = status
+            let success = data?[0] as! Bool
+            let message = data?[1] as! String
+            let error = data?[2] as! String
+            
+            
+            if (success) {
                 DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
                     self.openManagingSocket()
                 }
                 self.handler = handler
                 handler(.success(Void()))
-            } else {
-                self.coolDown()
+            }
+            else{
+                self.state = .disconnected
                 self.configURL = nil
-                handler(.failure(Error.helperRejected))
+                if(error == "virus" ){
```

Comment:
It is NOT a virus. Thees are potentially harmful commands, not malicious per se. Don't scare the user unnecessarily.